react_components/PaginationBoxView.js

```diff
@@ -61,6 +61,12 @@ export default class PaginationBoxView extends Component {
     }
   }
 
+  componentWillReceiveProps(nextProps) {
+    if (this.props.forceSelected !== nextProps.forceSelected) {
```

Comment:
Yes, you're right.
I forgot it, b/c in my case I am sure that `forceSelected` is always set, but this might not be the case for all users. I'll submit a fix!